### PR TITLE
Update 19

### PR DESCRIPTION
**Pull request by Rainb0wKey**
Soon part 2 of update (`$endif`/`$eleif[]`/Update of guide)

### CODE REVIEW

**Comment by caramellya-mei** [comment]
Aren't categorized functions supposed to be kept to [general guides](https://github.com/nilpointer-software/bdfd-wiki/tree/dev/src/guides/general) now?
If you made it so the users don't get thrown into 404 page by opening `$if[]` from the app, make a redirect to [ifStatements.md](https://github.com/NilPointer-Software/bdfd-wiki/blob/dev/src/guides/general/ifStatements.md) in book.toml instead

**Comment by Rainb0wKey** [comment]
I add all the functions from the guides to the functions section, in the guides you can find out how to use them, and on the pages you can find information about the function

**Comment by Rainb0wKey** [comment]
`Awaited Commands` have already been moved to the Functions category

File: src/bdscript/if.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# $if
+Returns `true` if provided condition is true, otherwise `false` is returned.
```
Comment (MineBartekSA) [inline src/bdscript/if.md]:
```suggestion
Executes the following block of code if the provided condition is true.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Review by MineBartekSA — APPROVED** [review]
Thank you for your hard work!

**Pull request merged**